tests/snapshots/TestCLICommands_atmos_non-existent.stderr.golden

```diff
@@ -1,3 +1,2 @@
-Error: unknown command "non-existent" for "atmos"
-Run 'atmos --help' for usage.
-unknown command "non-existent" for "atmos"
+Error: Unknown command "non-existent" for "atmos"
```

Comment:
We're missing:

```
Run 'atmos --help' for usage.
```

Reply:
ohhh, this went to `stderr`. 

I think errors should consistently go to `stderr`, and `stdout` be reserved for _expected_ output.

Reply:
Done